server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/WebSocketContract.scala

```diff
@@ -577,13 +577,137 @@ trait WebSocketContract {
     val mailboxState: String = jmapGuiceProbe.getLatestMailboxState(accountId).getValue.toString
 
     val mailboxStateChange: String = s"""{"@type":"StateChange","changed":{"$ACCOUNT_ID":{"Mailbox":"$mailboxState"}}}"""
-    val emailStateChange: String = s"""{"@type":"StateChange","changed":{"$ACCOUNT_ID":{"Email":"$emailState"}}}"""
+    val emailStateChange: String = s"""{"@type":"StateChange","changed":{"$ACCOUNT_ID":{"EmailDelivery":"$emailState","Email":"$emailState"}}}"""
 
     assertThat(response.toOption.get.asJava)
       .hasSize(3) // email notification + mailbox notification + API response
       .contains(mailboxStateChange, emailStateChange)
   }
 
+  @Test
+  @Timeout(180)
+  // For client compatibility purposes
+  def emailDeliveryShouldNotIncludeFlagUpdatesAndDeletes(server: GuiceJamesServer): Unit = {
```

Comment:
No test to check if we get a EmailDelivery notification after creating one?

Reply:
`dataTypesShouldDefaultToAll` already ensure that.

I'm unsure of the added value of a separated test.